amoskeyz/Epic-mail
# [Bug #164456126] Fix UI Pages

#### What does this PR do?
Fix UI Pages

#### Description of Task to be completed?
-make landing text appear clearly on the background image
-hand pointer should be on sign-up and sign-in button

#### How should this be manually tested?
-open your browser
-navigate to 'https://amoskeyz.github.io/Epic-mail/UI/index.html
- or navigate to 'https://amoskeyz.github.io/Epic-mail/UI/Sign-up.html
- you should be able to see the hand pointer on the sign-up, sign-in button
-text on the the landing page appears clearly because of  the changed background image

#### What are the relevant pivotal tracker stories?
<a href='https://www.pivotaltracker.com/story/show/164456126'>#164456126</a>
